### DISCUSSION TITLE
Should <sp-card>'s cover image's overflow be hidden?

### DISCUSSION THREAD

**Discussion by lehelen19** [adobe/spectrum-web-components#4190] [Q&A]
The `overflow: hidden` was removed in the [v0.35.0 upgrade](https://github.com/adobe/spectrum-web-components/commit/9cccd26f00f688cbe1477dc7a9ce01542b179737#diff-63a39db8f770105d87999ced7a0595689ffbf21fddefbe50ca04c9c043092e89L48), which can cause the image to peek out despite the card's `border-radius` being set.

**Comment by Westbrook** [comment]
This is a great question for @pfulton and the https://github.com/adobe/spectrum-css where we get these rules. It seems in their context a `.spectrum-Card-preview` will get this rule but a `.spectrum-Card-coverPhoto` won't. Could that be something we normalize Patrick?

**Reply by pfulton** [reply]
@lehelen19 @Westbrook do you have an example of where this is causing an issue? We were discussing this on our side, but we'd like to fully understand or see how it's causing problems so that we can be sure to provide the right answer(s). Overflow might not necessarily provide the right fix. (cc: @jawinn @mdt2)

**Reply by lehelen19** [reply]
An example is when we want to set the height, border-radius, and border-width of a card, and we want the image to be contained in the card. Setting `overflow: hidden` on `sp-card` would do the trick: https://studio.webcomponents.dev/edit/Z18P0UkZe89jaYBY8HVi/src/index.ts?p=stories
Is there a different approach I should consider, or is this a specified scenario enough to apply only sometimes?

**Reply by pfulton** [reply]
Ok, we've got this slated for our next sprint. Tracking it in [CSS-734](https://jira.corp.adobe.com/browse/CSS-734)

**Reply by jawinn** [reply]
I just wanted to note that this updated in the **8.0.0** version of Card in Spectrum CSS [with this Spectrum CSS PR](https://github.com/adobe/spectrum-css/pull/2661). So this issue looks like it will be resolved with the next release of SWC (using `"@spectrum-css/card": "^8.1.0"`).

**Reply by Westbrook** [reply]
Thanks for connecting the docs here. We should have this included in a release within the next week!
